contracts/Vault.vy

```diff
@@ -1323,17 +1360,21 @@ def _creditAvailable(strategy: address) -> uint256:
     # Adjust by the global debt limit left
     available = min(available, vault_debtLimit - vault_totalDebt)
 
-    # Adjust by the rate limit algorithm (limits the step size per reporting period)
-    delta: uint256 = block.timestamp - strategy_lastReport
-    # NOTE: Protect against unnecessary overflow faults here
-    # NOTE: Set `strategy_rateLimit` to 0 to disable the rate limit
-    if strategy_rateLimit > 0 and available / strategy_rateLimit >= delta:
-        available = strategy_rateLimit * delta
-
     # Can only borrow up to what the contract has in reserve
     # NOTE: Running near 100% is discouraged
-    return min(available, self.token.balanceOf(self))
-
+    available = min(available, self.token.balanceOf(self))
+
+    # Adjust by min and max borrow limits (per harvest)
+    # NOTE: min increase can be used to ensure that if a strategy has a minimum
+    #       amount of capital needed to purchase a position, it's not given capital
+    #       it can't make use of yet.
+    # NOTE: max increase is used to make sure each harvest isn't bigger than what
+    #       is authoirized. This combined with adjusting min and max periods in
+    #       `BaseStrategy` can be used to effect a "rate limit" on capital increase.
+    if available < strategy_minDebtPerHarvest:
+        return 0
+    else:
+        return min(available, strategy_maxDebtPerHarvest)
```

Comment:
this means that if I create a hegic vault, users deposit 888_000 * 10. I would need to do 10 harvests to fill a strategy.

Reply:
yes